code/typescript/pipelines-workshop/bin/cdk-workshop.ts

```diff
@@ -0,0 +1,5 @@
+import { App } from '@aws-cdk/core';
+import { CdkWorkshopStack } from '../lib/cdk-workshop-stack';
+
+const app = new App();
+new CdkWorkshopStack(app, 'CdkWorkshop');
```

Comment:
This required file was present in the main-workshop but not in pipelines-workshop.